Source/Core/Core/ConfigLoaders/NetPlayConfigLoader.cpp

```diff
@@ -39,6 +41,22 @@ class NetPlayConfigLayerLoader final : public Config::ConfigLayerLoader
 
     layer->Set(Config::SYSCONF_PROGRESSIVE_SCAN, m_settings.m_ProgressiveScan);
     layer->Set(Config::SYSCONF_PAL60, m_settings.m_PAL60);
+
+    if (m_settings.m_SyncSaveData)
```

Comment:
This can be joined as:

```cpp
if (m_settings.m_SyncSaveData && !m_settings.m_IsHosting)
```

unless extra code is planned to be added to the outer if statement body.

Reply:
There already is code in the outer if statement body. I'll add a line break to make it clearer.